examples/runners/wgpu/build.rs

```diff
@@ -4,6 +4,7 @@ use std::error::Error;
 fn build_shader(path_to_create: &str) -> Result<(), Box<dyn Error>> {
     SpirvBuilder::new(path_to_create)
         .spirv_version(1, 0)
+        .target("spirv-unknown-vulkan1.1")
```

Comment:
whoops, didn't hit "submit comment", I thought wgpu used vulkan 1.0?

Reply:
> I thought wgpu used vulkan 1.0?

I don't know, really. I tried googling to see if there was but couldn't find anything that mentions a minimum compatible version. I did run the wgpu example, so it definitely works at least.

Reply:
Currently the target doesn't actually affect the binary, only which validator is run (AFAIK), so specifying vulkan1.1 here will still produce a vulkan1.0 binary (due to `.spirv_version(1, 0)`), so it'll run on wgpu. It should still be `vulkan1.0`, though, I think.